src/exoplanet/orbits/simple.py

```diff
@@ -22,16 +23,23 @@ class SimpleTransitOrbit:
 
     """
 
-    def __init__(self, period=None, t0=0.0, b=0.0, duration=None, r_star=1.0):
+    def __init__(
+        self, period=None, t0=0.0, b=0.0, duration=None, r_star=1.0, ror=0
+    ):
         self.period = as_tensor_variable(period)
         self.t0 = as_tensor_variable(t0)
         self.b = as_tensor_variable(b)
-        self.duration = as_tensor_variable(duration)
+        if duration is not None:
+            self.duration = as_tensor_variable(duration)
+        else:
+            raise ValueError("`duration` must be provided.")
+
```

Comment:
`period` is also required. How about we change the function signature to:

```python
def __init__(self, *, period, duration, t0=0.0, b=0.0, r_star=1.0, ror=0.0):
```

and remove this check?

Reply:
Definitely happy to do this, I'll just have to update the tests because they're specified without kwarg names e.g `SimpleTransitOrbit(period, t0, b, r_star, duration)`. Are you happy if I do that?